lib/BuildSystem/SwiftTools.cpp

```diff
@@ -220,6 +223,11 @@ class SwiftCompilerShellCommand : public ExternalCommand {
     if (isLibrary) {
       result.push_back("-parse-as-library");
     }
+    if (enableWholeModuleOptimization) {
+      result.push_back("-whole-module-optimization");
+      result.push_back("-num-threads");
+      result.push_back("1");
```

Comment:
I think it is ok to leave them there, if it is convenient, but I think we need to be careful to only link the correct `.o` files (for which I think there will just be one).